src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -103,6 +105,16 @@ namespace AppInstaller::CLI
 
     void InstallCommand::ExecuteInternal(Context& context) const
     {
+        bool resumeExperimentalFeatureEnabled = Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::Resume);
+
+        if (resumeExperimentalFeatureEnabled && WI_IsFlagClear(context.GetFlags(), Execution::ContextFlag::Resume))
+        {
+            auto& checkpointManager = CheckpointManager::Instance();

```

Comment:
This code should not be in an individual command.  I think that the manager should live in the context (and certainly not be a singleton).